### PR TITLE
test(connector-besu): migrate v21-get-past-logs-endpoint to Jest

### PR DESCRIPTION
**Pull request by Udhayakumari**
test(connector-besu): migrate v21-get-past-logs-endpoint to Jest

Primary Changes:
```
packages/cactus-test-plugin-ledger-connector-besu/src/test/typescript/
integration/plugin-validator-besu/v21-get-past-logs-endpoint.test.ts
```
Fixes: #3572

Signed-off-by: Udhayakumari <62878965+Udhayakumari@users.noreply.github.com>

**Pull Request Requirements**
- [ ] Rebased onto `upstream/main` branch and squashed into single commit to help maintainers review it more efficient and to avoid spaghetti git commit graphs that obfuscate which commit did exactly what change, when and, why.
- [ ] Have git sign off at the end of commit message to avoid being marked red. You can add `-s` flag when using `git commit` command. You may refer to this [link](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) for more information.
- [ ] Follow the Commit Linting specification. You may refer to this [link](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification) for more information. 

**Character Limit**
- [ ] Pull Request Title and Commit Subject must not exceed 72 characters (including spaces and special characters).
- [ ] Commit Message per line must not exceed 80 characters (including spaces and special characters).

**A Must Read for Beginners**
For rebasing and squashing, here's a [must read guide](https://github.com/servo/servo/wiki/Beginner's-guide-to-rebasing-and-squashing) for beginners.

### CODE REVIEW

**Review by petermetz — CHANGES_REQUESTED** [review]
@Udhayakumari  +1 to what @outSH said, looking GREAT otherwise! Thank you very much!

**Comment by Udhayakumari** [comment]
Hello @outSH @petermetz, I have made changes you suggested. Please re-review and let me know!

**Review by petermetz — APPROVED** [review]
@Udhayakumari LGTM, thank you again for the contribution and the updates!

**Review by outSH — APPROVED** [review]

**Review by petermetz — CHANGES_REQUESTED** [review]
@Udhayakumari Please fix the linter errors. You can run it locally by doing `yarn lint` on the command line.

image

**Comment by Udhayakumari** [comment]
@petermetz I have fixed the linter error. Please re-review.

File: tools/ci-skip-for-maintainers.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import { readFileSync } from "fs";
```
Comment (petermetz) [inline tools/ci-skip-for-maintainers.js:1]: @Udhayakumari The linter is now fixed, thank you for that! On another note though, this file shouldn't  be in here, did you maybe accidentally add it during a rebase? Either way, please remove it from the diff and then we are good to go from my side.

**Review by petermetz — CHANGES_REQUESTED** [review]

**Review by petermetz — APPROVED** [review]

**Review by petermetz — CHANGES_REQUESTED** [review]
@Udhayakumari Backtracking again: the linter still fails like before so it needs to be fixed first before we can merge. Let me know if you need help:

image

**Comment by Udhayakumari** [comment]
@petermetz , I have fixed the linter error. Please re-review.

linter-error

**Review by petermetz — APPROVED** [review]
@Udhayakumari LGTM, thank you!

**Pull request merged**